src/sentry/api/endpoints/project_metrics.py

```diff
@@ -50,22 +53,26 @@ class ProjectMetricsVisibilityEndpoint(ProjectEndpoint):
 
     def _handle_by_operation_type(
         self, request: Request, project: Project, metric_operation_type: MetricOperationType
-    ):
+    ) -> MetricBlocking:
         metric_mri = request.data.get("metricMri")
         if not is_mri(metric_mri):
             raise InvalidParams("You must supply a valid metric mri")
 
         metric_mri = cast(str, metric_mri)
+        patched_metrics: Mapping[int, MetricBlocking] = {}
+
         if metric_operation_type == MetricOperationType.BLOCK_METRIC:
-            block_metric(metric_mri, [project])
+            patched_metrics = block_metric(metric_mri, [project])
         elif metric_operation_type == MetricOperationType.UNBLOCK_METRIC:
-            unblock_metric(metric_mri, [project])
+            patched_metrics = unblock_metric(metric_mri, [project])
         elif metric_operation_type == MetricOperationType.BLOCK_TAGS:
             tags = request.data.get("tags") or []
-            block_tags_of_metric(metric_mri, set(tags), [project])
+            patched_metrics = block_tags_of_metric(metric_mri, set(tags), [project])
         elif metric_operation_type == MetricOperationType.UNBLOCK_TAGS:
             tags = request.data.get("tags") or []
-            unblock_tags_of_metric(metric_mri, set(tags), [project])
+            patched_metrics = unblock_tags_of_metric(metric_mri, set(tags), [project])
+
+        return patched_metrics[project.id]
```

Comment:
Since for now we support just one project, I just return the patched metric of that project.